cetmix_tower_server/wizards/__init__.py

```
@@ -1,4 +1,6 @@
+from . import cx_tower_command_run_wizard_variable_value
```

Flake8 (7.2.0)

[error] '.cx_tower_command_run_wizard_variable_value' imported but unused

(F401)